tsconfig.json

```diff
@@ -13,6 +13,7 @@
     "esModuleInterop": true,
     "outDir": "dist/lib",
     "declarationDir": "dist/types",
+    "skipLibCheck": true,
```

Comment:
according to https://www.typescriptlang.org/tsconfig#skipLibCheck we would loose type safety here. Can we drop it?